src/uu/chgrp/src/chgrp.rs

```diff
@@ -140,24 +126,12 @@ pub fn uu_app() -> Command {
                 .long(options::RECURSIVE)
                 .help("operate on files and directories recursively")
                 .action(ArgAction::SetTrue),
-        )
-        .arg(
-            Arg::new(options::traverse::TRAVERSE)
-                .short(options::traverse::TRAVERSE.chars().next().unwrap())
-                .help("if a command line argument is a symbolic link to a directory, traverse it")
-                .action(ArgAction::SetTrue),
-        )
-        .arg(
-            Arg::new(options::traverse::NO_TRAVERSE)
-                .short(options::traverse::NO_TRAVERSE.chars().next().unwrap())
-                .help("do not traverse any symbolic links (default)")
-                .overrides_with_all([options::traverse::TRAVERSE, options::traverse::EVERY])
-                .action(ArgAction::SetTrue),
-        )
-        .arg(
-            Arg::new(options::traverse::EVERY)
-                .short(options::traverse::EVERY.chars().next().unwrap())
-                .help("traverse every symbolic link to a directory encountered")
-                .action(ArgAction::SetTrue),
-        )
+        );
+
+    // Add common arguments with chgrp, chown & chmod
+    for arg in uucore::perms::common_args() {
+        cmd = cmd.arg(arg);
+    }
+
+    cmd
```

Comment:
This is not necessary. `clap` provides an `args` function for adding multiple args and so you can use: `.args(uucore::perms::common_args())`. It also allows you to get rid of the `cmd` var.

Reply:
Much better, thanks :)